server/datastore/mysql/migrations/tables/20240705152704_AddHostUuidToMdmIdpAccounts.go

```diff
@@ -0,0 +1,42 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240705152704, Down_20240705152704)
+}
+
+func Up_20240705152704(tx *sql.Tx) error {
+	alterStmt := `
+ALTER TABLE mdm_idp_accounts ADD COLUMN (
+  host_uuid varchar(63) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+  fleet_enroll_ref varchar(63) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT ''
```

Comment:
Any reason for `63`? UUIDs are generally 36 chars long I think, and it looks like we often declare the column as 255 in other tables.

Reply:
@mna, no particular reason. I saw `63` used in connection with another UUID field in our code as well as other values you mentioned. I know we've started adding UUID prefixes in some cases so it seemed like it gave some room more than `36` but short of `255` (which I also considered but hesitated because we've run into the max composite key length issues in other places, although we're a long ways away from that with this table). 

Totally open to going with another approach. What do you think is a good pattern for us to use generally for UUIDs? 255 or something smaller?

Reply:
If we use 63 elsewhere, I think that's fine too. Probably something we should've been consistent with earlier on, but we can pick that size and starts using it. If we were super-totally sure of the content of those fields (eg. adding a new field that we fully control the value) I'd say let's go with 36, but I don't know how safe that is for what we store in `host_uuid` and the enroll reference and things like that, so a little wiggle room might be a good idea for those cases!